### DISCUSSION TITLE
[Bug] 连按两次“回退三秒”快捷键后，视频画面卡顿、音频重复播放回退部分

### DISCUSSION THREAD

**Discussion by sharkucn** [ywmoyue/biliuwp-lite#777] [bug]
### Bug 描述

之前版本的连按两次触发自动回退的现象消失

现版本在视频播放时连按两次“回退三秒”快捷键（设置为“方向键左”），有概率触发：
视频卡顿（向后继续播放）、音频流畅（重复播放回退的6s内容）

另有：从历史记录打开有播放进度但未播完的视频，选择开始播放后（快捷键或点击画面右下小电视）视频没反应（进度条不变，实际未开始播放），刷新页面后恢复正常

### 操作步骤

```plain text
1. 打开应用
2. 随意点开一个视频
3. 播放一段时间后连按两次“回退三秒”快捷键
4. 概率触发Bug

2a. 从历史记录打开一个未看完的视频
3a. 开始播放，触发bug
```

### 期望结果

能够正常播放

### 截图

回退Bug触发后的效果（我的OBS录制开始快捷键为Ctrl+Z，结束为Ctrl+X，所以有切换分P弹窗）
https://github.com/user-attachments/assets/c3445001-06c6-48bc-b600-758b17fbe17d

### 日志

[20240819_Bug1.log](https://github.com/user-attachments/files/16670522/20240819_Bug1.log)
[20240820_Bug1.log](https://github.com/user-attachments/files/16670524/20240820_Bug1.log)
[20240820_Bug2.log](https://github.com/user-attachments/files/16670525/20240820_Bug2.log)

Bug1指双击回退Bug，Bug2指已有进度续播Bug

### 应用来源

Github/其他Github镜像站

### 安装方式

脚本安装

### 系统版本

其它 (需注明)

### 其它系统版本

Windows 11 27686.1000

### 设备类型

笔记本

### 设备架构

x64

### 应用架构

x64

### 备注

软件安装方式：卸载旧版本后，脚本全新安装。版本号4.7.1.202
考虑到Windows版本（Canary），可能问题不出在软件上  :(

### 代码贡献

我更擅长发现问题

**Comment by sharkucn** [comment]
修正：任意次数的连续快进/快退都可能触发

**Comment by ywmoyue** [comment]
4.7.2 已处理
